### PR TITLE
 update CONTRIBUTING with details on release process

### PR DESCRIPTION
**Pull request by Chris-Hibbert**
Add detail to CONTRIBUTING.md on the life cycle of proposals, from `a3p-integration`, through releases to `agoric-3-proposals.

### CODE REVIEW

**Comment by Chris-Hibbert** [comment]
cc: @mhofman

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -2,8 +2,18 @@
 
 Each proposal to Mainnet (agoric-3) should be contributed to this repo before being proposed to the chain.
 
-1. Prepare the proposal and verify it running locally
+1. Prepare the proposal and verify it by creating a proposal in
```
Comment (turadg) [inline CONTRIBUTING.md:5]: please link to it. it's stable enough: https://github.com/Agoric/agoric-sdk/blob/master/a3p-integration/README.md

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -2,8 +2,18 @@
...
+   agoric-sdk/a3p-integration. See the README there for more details
+
 2. Submit a PR to this repo in which the proposal has a _pending_ name (e.g. `a:my-contract`)
+
 3. Get review and merge to main
+
 4. When the proposal has been further qualified with higher fidelity to Mainnet, submit to Mainnet
-5. When the proposal passes, submit a PR here converting the pending proposal to a passed one (by changing the letter to the submission number)
+
+5. When the proposal passes, submit a PR here converting the pending proposal to a passed one (by
```
Comment (turadg) [inline CONTRIBUTING.md:14]: this is showing as an edit but I don't see the change.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -2,8 +2,18 @@
...
+   changing the letter to the submission number)
+
+6. Remove the proposal from a3p-integration. Move generally useful tools to synthetic-chain
```
Comment (turadg) [inline CONTRIBUTING.md:17]:
```suggestion
6. Remove the proposal from a3p-integration. Move generally useful tools to the `@agoric/synthetic-chain` package
```

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -2,8 +2,18 @@
...
+   here. Move tests that should continue to pass afterwards into a3p-integration/z:acceptance
```
Comment (turadg) [inline CONTRIBUTING.md:18]: Minor, but the latter sentences need to be done before the first. consider sub-bullets or moving this process info to a3p-integration.

File: CONTRIBUTING.md (hunk 1 above)
Comment (Chris-Hibbert) [reply CONTRIBUTING.md:5]: done

File: CONTRIBUTING.md (hunk 2 above)
Comment (Chris-Hibbert) [reply CONTRIBUTING.md:15]: Added a line break.

File: CONTRIBUTING.md (hunk 4 above)
Comment (Chris-Hibbert) [reply CONTRIBUTING.md]: re-ordered and converted to sub-bullets.

File: CONTRIBUTING.md (hunk 3 above)
Comment (Chris-Hibbert) [reply CONTRIBUTING.md]: done

**Pull request merged**